pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -182,7 +182,8 @@
      * 1 second.
      *
      * <p>By default, the acknowledge timeout is disabled and that means that messages delivered to a
-     * consumer will not be re-delivered unless the consumer crashes.
+     * consumer will not be re-delivered unless the consumer crashes. Since 2.3.0, when a dead letter policy
+     * is specified, and no ackTimeoutMillis is specified, then the ack timeout will be set to 30 seconds.
```

Comment:
```suggestion
     * is specified, and no ackTimeoutMillis is specified, the ack timeout is set to 30 seconds.
```
The present tense is preferred.